src/rust/engine/fs/store/src/cli_options_tests.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Nit: Idiomatic Rust would put this into the file under test in a sub-module via:

```rust
#[cfg(test)]
mod tests {
 ...
}
```

But for Pants we put them into separate files.

No change needed per se. Just calling out our difference in idiomatic Rust.

Reply:
Yeah, but I figure it's best to be locally consistent. TBH I also like what we do, since it makes it easy to see which modules don't have tests (of course ideally they all would, but...)